packages/devtools_app/lib/src/primitives/trees.dart

```diff
@@ -130,13 +128,13 @@ abstract class TreeNode<T extends TreeNode<T>> {
     return children.removeAt(index);
   }
 
-  void addAllChildren(List<T> children) {
-    children.forEach(addChild);
+  void addAllChildren(List<T?> children) {
+    children.forEach(addChild as void Function(T?));
```

Comment:
can we drop the `?` marks here? I don't know why they'd be necessary since `addChild` takes a non nullable `T child`